theme/fonts.js

```diff
@@ -25,8 +25,7 @@ export const LINEHEIGHTS = {
 }
 
 export const FONTWEIGHT = {
-  base: '400',
-  bold: '800',
+  base: '200',
```

Comment:
can we remove it entirely?

Reply:
yes, I was originally going to keep using it to reset the checkbox label weight but I ended up removing it in the end